autobuild/windows/autobuild_windowsinstaller_1_prepare.ps1

```diff
@@ -19,25 +19,31 @@ $ErrorActionPreference = "Stop"
 ###  PROCEDURE  ###
 ###################
 
+$QtDir = 'C:\Qt'
+$Qt32Version = "5.15.2"
+$Qt64Version = "5.15.2"
+$AqtinstallVersion = "2.0.5"
+$JackVersion = "1.9.17"
+
 echo "Install Qt..."
 # Install Qt
-pip install aqtinstall
+pip install "aqtinstall==$AqtinstallVersion"
 if ( !$? )
 {
 		throw "pip install aqtinstall failed with exit code $LastExitCode"
 }
 
 echo "Get Qt 64 bit..."
 # intermediate solution if the main server is down: append e.g. " -b https://mirrors.ocf.berkeley.edu/qt/" to the "aqt"-line below
-aqt install-qt --outputdir C:\Qt windows desktop 5.15.2 win64_msvc2019_64
+aqt install-qt --outputdir "${QtDir}" windows desktop "${Qt64Version}" win64_msvc2019_64
 if ( !$? )
 {
 		throw "64bit Qt installation failed with exit code $LastExitCode"
 }
 
 echo "Get Qt 32 bit..."
 # intermediate solution if the main server is down: append e.g. " -b https://mirrors.ocf.berkeley.edu/qt/" to the "aqt"-line below
-aqt install-qt --outputdir C:\Qt windows desktop 5.15.2 win32_msvc2019
+aqt install-qt --outputdir "${QtDir}" windows desktop "${Qt32Version}" win32_msvc2019
```

Comment:
msvc 2019 should also be in a variable

Reply:
Done. I've moved 2019 to a `$MsvcVersion` variable.